### DISCUSSION TITLE
Proposal: create a help command

### DISCUSSION THREAD

**Discussion by samsmithnz** [isaacrlevin/WorkFlowGenerator#17] [Ideas]
**Purpose:** create self-documentation using typically used help commands.

for example
```
dotnet-workflow-generator --help
```

help aliases to consider: 
```
[-h|--help|-?]
```

Output would be similar to dotnet - [listing all parameters similar to it's documentation](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build).

**Comment by isaacrlevin** [comment]
Right now there are no params to the command but this might be useful in the future
